lfsapi/client.go

```diff
@@ -130,7 +130,16 @@ func (c *Client) doWithRedirects(cli *http.Client, req *http.Request, via []*htt
 
 	c.traceResponse(req, tracedReq, res)
 
-	if res.StatusCode != 307 {
+	if res.StatusCode != 301 &&
+		res.StatusCode != 302 &&
+		res.StatusCode != 303 &&
+		res.StatusCode != 307 &&
+		res.StatusCode != 308 {
```

Comment:
Thanks 👍. I don't have strong feelings either way and don't want to reach for a map/func until we need to share this check somewhere else.